### ISSUE TITLE
Model not learning

### ISSUE THREAD

**Issue by SS8188** [mrdbourke/pytorch-deep-learning#889]
Timestamp  7:50:00
PyTorch workflow section exercise
Issue:My model is not updating its parameters(weight & bias) to minimize the Loss when I run the training loop.(Note that I have created training & testing split using shuffling of indices instead of straight away slicing the dataset into 2 parts)
Notebook:https://colab.research.google.com/drive/1vg0yCPaxVm_nOE9Vke9pRxy299LXEicC?usp=drive_link

I tried setting the random_seed value to the same number(42) everywhere,The learning rate is set to 0.01(mentioned in the exercise),I tried changing it but I am not able to fix this(I did the saving & loading part of the exercise mindlessly even when the model isn't working).
